fiat-plugins-test/fiat-plugins-test.gradle

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Open to breaking out the plugin TCK into a separate PR but wanted to be sure everything worked all the way through.

Reply:
No - this makes the PR complete imo. Nice.